core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.template.html

```diff
@@ -108,10 +118,10 @@ <h3>Upload Thumbnail</h3>
   .oppia-thumbnail-preview-description {
     font-size: 16px;
     line-height: 1.2;
-    max-height: 92px;
     overflow: hidden;
     padding: 0 16px 0 16px;
-    word-break: break-word;
+    text-overflow: ellipsis;
+    white-space: nowrap;
```

Comment:
Don't we want to wrap for long titles containing spaces?

Reply:
Yep, this handles that. PTAL screenshots [here](https://github.com/oppia/oppia/pull/9216#issue-412545642)!

Reply:
Hmm, but I would assume that "Really really really really really really long title" was formated ± this way:
```
Really really really really
really really long title
```

and not

```
Really really really really really reall…
```

Reply:
Multiple lines do not fit in the given space. Besides we will be changing this later on as part of the editor redesign project, perhaps it can be handled better then.

Reply:
Also, this seems to be expectation in the original issue (see [here](https://github.com/oppia/oppia/issues/9209#issue-611248783)).

Reply:
I don't have a strong opinion about this, and sure it is okay to keep it like this for now.